### DISCUSSION TITLE
question: clip skip

### DISCUSSION THREAD

**Discussion by kalle07** [lllyasviel/Fooocus#3023] [Q&A] [answered]
why some of models dont work with clip-skip 1 ... only 2
its appears a dark noisy image
with clip 2 it works ... so lala, some times strukturtes and monsters appear, never seen that in a1111
in automatic it works with 1   ;)

**Comment by ILikeToasters** [comment, marked as answer]
CLIP Skip at 2 is the default and usually the best option but this gives you the ability to change it if you want.  Before having the option to change, 2 was what it was set at previously.

**Reply by kalle07** [reply]
yes... but why work clipskip 1 in a1111 and in fooocus dont ? ^^

**Reply by Madxis** [reply]
Using the same model/checkpoint?

**Reply by mashb1t** [reply]
See description of https://github.com/lllyasviel/Fooocus/pull/3011, it does work, but just a bit differently. Same logic for ComfyUI as in Fooocus btw.

**Reply by kalle07** [reply]
yes, same checkpoint.
i tested 10 models ... half of them produce a dark noisy images on clip 1 in a1111 all is working.
clip 2 often the weights of loras very different i think?!?
